config/videos.yml

```diff
@@ -0,0 +1,12 @@
+"ChA_zph7aao":
```

Comment:
I thought about pulling from the Youtube API. However since these don't have all that often, and that would an extra dependency I opted to hard code the data.